pkg/services/login/authinfo.go

```diff
@@ -46,7 +46,9 @@ func GetAuthProviderLabel(authModule string) string {
 		return "JWT"
 	case AuthProxyAuthModule:
 		return "Auth Proxy"
+	case "oauth_generic_oauth":
+		return "Generic OAuth"
```

Comment:
```suggestion
		return "OAuth"
```

Reply:
so far we follow the configuration "pattern" of the [auth.provider] and then return that value. This would be the "outlier" for that pattern

Thought this would be good for the implementation when we want to have this more configured. I guess this is not really concerning as it is to the frontend.

Reply:
let's revisit this when we refactor!